play/src/front/Phaser/Game/GameScene.ts

```diff
@@ -1672,8 +1672,8 @@ ${escapedMessage}
             iframeListener.setTilesStream.subscribe((eventTiles) => {
                 for (const eventTile of eventTiles) {
                     this.gameMapFrontWrapper.putTile(eventTile.tile, eventTile.x, eventTile.y, eventTile.layer);
+                    this.animatedTiles.updateAnimatedTiles(eventTile.x, eventTile.y);
```

Comment:
@Waclaw-I We should maybe put the call to `this.animatedTiles.updateAnimatedTiles` inside `this.gameMapFrontWrapper.putTile` ?